src/core/server/legacy_compat/__snapshots__/legacy_service.test.ts.snap

```diff
@@ -37,7 +37,7 @@ Array [
           ],
           "results": Array [
             Object {
-              "isThrow": false,
+              "type": "return",
```

Comment:
Jest internals here as well, though this may be a bit hard to fix since it's an entire object. I'm not sure why we're testing such large snapshots like this though.